scripts/telegraf.sh

```diff
@@ -15,3 +15,21 @@ start_telegraf() {
 start_host_metrics_telegraf() {
     telegraf --config "$(pwd)/telegraf/telegraf.host.conf"
 }
+
+# Import metrics in file defined by $HOLOCHAIN_INFLUX_FILE into InfluxDB instance via telegraf.
```

Comment:
I'm not following why this function is needed. It's looking at the `HOLOCHAIN_INFLUX_FILE` variable but that might be set in a different shell where Holochain was run. Then it's trying to upload metrics itself with Telegraf but the Nix script it's being called from should do that job for all metrics?